doc/build-unix.md

```diff
@@ -98,13 +99,23 @@ install necessary parts of boost:
 BerkeleyDB is required for the wallet.
 
 **For Ubuntu only:** db4.8 packages are available [here](https://launchpad.net/~bitcoin/+archive/bitcoin).
+
 You can add the repository and install using the following commands:
 
+**For Ubuntu 18.04 and earlier versions**
+
     sudo apt-get install software-properties-common
     sudo add-apt-repository ppa:bitcoin/bitcoin
     sudo apt-get update
     sudo apt-get install libdb4.8-dev libdb4.8++-dev
 
+**For Ubuntu 20.04+**
+
+    sudo apt-get install software-properties-common
+    sudo apt-add-repository "deb http://ppa.launchpad.net/bitcoin/bitcoin/ubuntu cosmic main"
+    sudo apt-get update
+    sudo apt-get install libdb4.8-dev libdb4.8++-dev
+
```

Comment:
Is it ok to recommend using an older repository? I seem to remember this causing issues in the past for other packages.

There's an install_db4.sh script in the contrib directory that we could suggest as an alternative, at least until we can standardize on BDB 5.3.

Reply:
@iFoggz I think @cyrossignol is right on this.